torch_xla/core/dynamo_bridge.py

```diff
@@ -230,16 +230,19 @@ def extract_graph_helper(xla_model: torch.fx.GraphModule):
       for xla_arg in xla_args
   ]
 
-  args_tensor_ids = [
-      torch_xla._XLAC._xla_get_tensor_id(xla_arg) for xla_arg in xla_args
-  ]
+  xla_tensor_args = [(i, xla_arg)
```

Comment:
naming is a bit misleading here, it is more of `arg_idx_and_xla_tensor_args`... open to other suggestions but I think we wan to make it clear that it is a list of `[idx, xla_arg]` tuple

Reply:
What about: `index_and_xla_tensor_args`?

Reply:
sgtm